test/apparmor.bats

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+    cleanup_test
+}
+
+# 1. test running with loading the default apparmor profile.
+# test that we can run with the default apparomr profile which will not block touching a file in `.`
+@test "load default apparomr profile and run a container with it" {
```

Comment:
@mrunalp yes, I think we should do this. And should we do the same thing in seccomp test?